scripts/osfstorage/usage_audit.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+TEMPDIR=`mktemp -d`
+trap "rm -rf $TEMPDIR" EXIT
+
+export HOME=$TEMPDIR
+cd /opt/apps/osf
+source /opt/data/envs/osf/bin/activate
```

Comment:
This was all copied from `scripts/osfstorage/glacier_audit.sh`
The shell scripts are used for cron jobs.